### DISCUSSION TITLE
Reduntant copy for the text options and display

### DISCUSSION THREAD

**Discussion by augustin-colesnic** [ezra-bible-app/ezra-bible-app#516] [General]
![Screenshot 2021-05-22 at 23 03 18](https://user-images.githubusercontent.com/592416/119239590-c878bb80-bb52-11eb-9011-e29c1d9c18ac.jpg)

The name Show is redundant in too many options so a better way would be to amend the copy with 'Show text options' and remove the Show word from Display section

**Comment by tobias-klein** [comment]
I just took a look at how Visual Studio Code does this ... and they have a similar approach to these kind of texts. Would be interesting to check some more apps. I am not against this, but if there is a common pattern with other apps we should stick to that I think.

![Bildschirmfoto 2021-05-23 um 21 12 31](https://user-images.githubusercontent.com/17739895/119273605-f8d06080-bc0b-11eb-9616-716651f0c218.png)

**Comment by augustin-colesnic** [comment]
That menu from VCode is not a good example. Ezra Bible App has a panel in this case, a section with a title and the SHOW word could be simplified as in the screenshot i attached in the begging of this issue. 

A good article to help with the decision would be this one:  https://www.nngroup.com/articles/reduce-redundancydecrease-duplicated-design-decisions/

**Comment by zhuiks** [comment]
Thanks for the article. 
I would say that in case of VSCode it's "Alternative Navigation Paths". As a user you can show/hide most of the thing without going into menu with 3-clicks.

I think the problem is a bit bigger than just using a different wording in the Options dialog. I think the dialog is very overloaded with options (and it's increasing) and as a new user it's very hard to figure out what those options are doing. You have to click a couple of times to figure out what that option does. 
In my opinion some options can be moved to "closer" to user like notes and tags options can be switchable buttons inside Bible browser and sidebar/dictionary can be buttons on the left like in VSCode (I mentioned that in #299 discussion). Option dialog itself can be transformed to a popover like Opera settings. That should be pretty easy refactoring as it's just changing the shape and position of the dialog. We can discuss all of that when we will talk on Skype. I'll try to enhance my sketch a bit more and post it in #299.
